repos/fdbt-site/src/pages/api/manageOperatorDetails.ts

```diff
@@ -52,7 +53,6 @@ export const collectErrors = (operatorDetails: OperatorDetails): ErrorInfo[] =>
     if (operatorDetails.email && !checkEmailValid(operatorDetails.email)) {
         errors.push({ id: 'email', errorMessage: 'Provide a valid email' });
     }
-
     if (operatorDetails.url && !/^[^ ]+$/.exec(operatorDetails.url)) {
```

Comment:
This is where we should remove `!/^[^ ]+$/.exec(operatorDetails.url)` and replace with your function.